templates/user/profile.html

```diff
@@ -43,8 +43,9 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   <article class=preview>
     <h1><span class="newtext title">&nbsp;</span></h1>
     <p><span class=newtext>&nbsp;</span><span class=newtext>&nbsp;</span><span class=newtext>&nbsp;</span></p>
-    <a href={{ url_for('article_create') }} class=read>Write this article</a>
+    <a href={{ url_for('article_create') }} class=read>Write an article</a>
```

Comment:
I'm still curious about testing the phrase _Write this article_. I won't mind at all changing it if it actually confuses people.